data/posts/rsc-framework-mode-preview.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "React Router RSC Framework Mode Preview"
+summary: React Router's preview support for RSC Framework Mode is now available
+date: 2025-09-18
+image: /blog-images/headers/rr-rsc-path-forward.jpg
+ogImage: /blog-images/headers/rr-rsc-path-forward.jpg
+imageAlt: Glowing tree
+imageDisableOverlay: true
+authors:
+  - Mark Dalgleish
+---
+
+Recently we shipped a [preview of React Router with support for React Server Components (RSC) in Data Mode](./rsc-preview). With the release of React Router v7.10, we're excited to announce that preview support for [RSC](https://react.dev/reference/rsc/server-components) is now also available in Framework Mode.
```

Comment:
The linked article wasn't for a Data Mode preview specifically, but switching the link to the Data Mode one doesn't really make sense either.

I think we should include both articles and clarify it like the following. Happy to hear other suggestions.

```suggestion
Recently we shipped a [preview of React Router with support for React Server Components (RSC)](./rsc-preview) as well as low-level APIs for [RSC support in Data Mode](./react-router-and-react-server-components). With the release of React Router v7.10, we're excited to announce that preview support for [RSC](https://react.dev/reference/rsc/server-components) is now also available in Framework Mode.
```